workspaces/ui-v2/src/lib/Interfaces.ts

```diff
@@ -30,12 +30,10 @@ export interface IPatchChoices {
 
 export interface IInteractionPreviewTab {
```

Comment:
allowsExpand and ignoreRule aren't used at all in the code. It might be missed during porting or it's never used.

Either way, I think removing this simplifies it and if we need to reimplement it, these could have used rewrites anyway (it's hard to trace what they were intended to do - probably copy paste from old optic iterations)